core/templates/dev/head/app.js

```diff
@@ -364,6 +364,17 @@ oppia.factory('validatorsService', [
           return false;
         }
         return true;
+      },
+      isValidExplorationId: function(input, showWarnings) {
+        // Exploration IDs are urlsafe base64-encoded.
+        var VALID_ID_CHARS_REGEX = /^[a-zA-Z0-9_\-]+$/g;
+        if (!input || !VALID_ID_CHARS_REGEX.test(input)) {
+          if (showWarnings) {
```

Comment:
Do we want to differentiate the warnings here? (ie. Valid vs blank, since we already have that info).